internal/pkg/merge/merger.go

```diff
@@ -227,16 +227,19 @@ func mergeDocumentType(base *sbom.DocumentType, t proto.Message) error {
 		return fmt.Errorf("%w: %T", errCastFailure, t)
 	}
 
-	if (base.Name == nil || *base.Name == "") && other.Name != nil {
-		base.Name = other.Name
+	if base.GetName() == "" && other.GetName() != "" {
+		name := other.GetName()
+		base.Name = &name
```

Comment:
I'm assuming not since you didn't do it, but might as well ask:
Can we not do?
```suggestion
		base.Name = &other.GetName()
```

Reply:
Yeah I did try that. Go doesn't let you address a value, only a variable. I assume it would be similar to trying to write `&"some string"`